src/components/PresentationThumb.vue

```diff
@@ -40,9 +40,9 @@
     },
     computed: {
       thumb_url: function () {
-        return 'http://i3.ytimg.com/vi/' + this.presentation.url + '/0.jpg'
-        // return 'http://i3.ytimg.com/vi/' + this.presentation.youtube_id + '/hqdefault.jpg'
-        // return 'http://i3.ytimg.com/vi/' + this.presentation.youtube_id + '/maxresdefault.jpg'
+        // return 'https://i3.ytimg.com/vi/' + this.presentation.url + '/0.jpg'
```

Comment:
similar to the earlier comment, it may be a good idea to pull domain names out to a config.js file. I feel a bit less strong about external domains, though the same easy-change logic still applies.